### ISSUE TITLE
Customizable removeElem() delay

### ISSUE THREAD

**Issue by returnlytom** [drozhzhin-n-e/ng2-tooltip-directive#2]
Before 1.1.2, the removeElem fired instantly; now there's a 300ms delay: 

![gif](http://g.recordit.co/jo54U6qFeD.gif)

Potential solution could be adding another attribute property to the directive. For example:

```html
<div tooltip="Tooltip" placement="left" remove-delay="100"></div>
```

**Comment by drozhzhin-n-e** [comment]
This is fixed in the 1.2.0 version
`<div tooltip="Tooltip" placement="left" show-delay="300" hide-delay="300"></div>`

**Comment by returnlytom** [comment]
Thanks @drozhzhin-n-e
